gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -470,7 +614,13 @@ public void setLocationLockEnabled(boolean enabled) {
 
   public enum Mode {
     DEFAULT,
-    REPOSITION
+    REPOSITION,
+    ADD_POLYGON
+  }
+
+  public enum PolygonDrawing {
+    DEFAULT,
```

Comment:
What does DEFAULT mean? And PolygonDrawing? Please choose more appropriate names and document.

Reply:
changed to STARTED